matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -1183,14 +1170,13 @@ var PhenoTips = (function (PhenoTips) {
 
     _afterProcessTableHideApsentServerIdsFromFilter : function()
     {
-        if (this._presentServerIds.length > 1) {
-            $('checkbox-server-filters').show();
-            $('matching-filters').select('input[type="checkbox"][name="checkbox-server-id-filter"]').each( function(selectEl) {
-                (this._presentServerIds.indexOf(selectEl.value) >= 0) ? selectEl.up().show() : selectEl.up().hide();
-            }.bind(this));
-        } else {
-            $('checkbox-server-filters').hide();
-        }
+        $('checkbox-server-filters').show();
```

Comment:
This line should be removed, together with line 90 which hides the element in the first place `$('checkbox-server-filters').hide()` since they just cancel each other out.